src/core/Pagination/index.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { forwardRef } from "react";
+import Icon from "../Icon";
+import MenuItem from "../MenuItem";
+import {
+  Page,
+  PaginationExtraProps,
+  StyledPagination,
+  StyledPaginationButtonIcon,
+  StyledPaginationChevronButton,
+  StyledPaginationDropdownMenu,
+} from "./style";
+import { usePagination } from "./usePagination";
+
+export interface PaginationInternalProps {
+  onPageChange: (page: number | string) => void;
+  onNextPage: () => void;
+  onPreviousPage: () => void;
+  totalCount: number;
+  siblingCount?: number;
```

Comment:
I'm not sure what siblingCount is meant to be ...

Reply:
is this like, how many thing are shown in the pagination bar or something?

Reply:
ooh yea, that will be a tough one to rename. I'm ok with leaving it if that's what mui uses!